src/Helpers/Form/Utils.php

```diff
@@ -83,3 +82,46 @@ function getFailedTransactionPageURL( $args = array() ) {
 	);
 }
 
+/**
+ * Get shortcode argument.
```

Comment:
Please provide a description of _when_ this function ought to be used. What context is it assuming?

Reply:
fixed: https://github.com/impress-org/givewp/compare/fb342b64d1a7...b03792348c28